src/gulpfile.js

```diff
@@ -271,6 +271,7 @@ gulp.task('build-mseditor', ['build-scripts-template-editor', 'add-revision-numb
 
 gulp.task('build-scripts-template-editor', function () {
   return gulp.src([
+    paths.app + '/env/' + process.env.NODE_ENV + '.js',
```

Comment:
I understand that you were having issues with scripts sorting inside the bundle, what happens if you add env entry after *.js entry?

Reply:
Oh that was a mistake, the order is the same as the declared order.

Reply:
In that case, you could include environment at the end, and you could avoid inline script in the HTML file.

Reply:
Hi andres, you mean to move:
```js
<script>
  // Create dopplerRelay module out of the scope of webapp to get environment constants
  angular.module('dopplerRelay', []);
</script>
```
inside of serviceAdapter at the top and move the js file with the constant definition to the bottom?

Reply:
Righ, inside the serviceAdapter or another .js file